course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -200,13 +200,26 @@ describe('Unit testing PeopleController', function() {
         beforeEach(setupController);
 
         it('won\'t add person if they already have a course enrollment', function(){
+            var newPerson = [
+                {
+                    active: 1,
+                    email_address: "test45@mcelroy.org",
+                    name_first: "Test",
+                    name_last: "User",
+                    prime_role_indicator: "Y",
+                    role_type_cd: "STUDENT",
+                    univ_id: "12345678"
+                }
+                ["test45@mcelroy.org"]
+            ];
             spyOn(scope, 'filterSearchResults').and.returnValue([personResult[0]]);
```

Comment:
I think these larger changes to this file can be avoided if we just change this one line to 
```
spyOn(scope, 'filterSearchResults').and.returnValue(personResult);
```